### ISSUE TITLE
[Hortolândia - SP ]Pesquisador de dados Senior na IBM

### ISSUE THREAD

**Issue by GiovanaNp1** [minas-ti/vagas#115]
### Seu papel e responsabilidades

Combina dados profundos e habilidades analíticas com forte perspicácia de negócios para resolver problemas de gerenciamento de TI e de negócios por meio da compreensão, preparação e análise de dados para prever tendências emergentes e fornecer recomendações para otimizar os resultados. As responsabilidades incluem:

- Analisar a qualidade e precisão dos dados e fornecer recomendações para melhorar sua confiabilidade.
- Trabalhar com líderes de negócios para resolver problemas de negócios, entendendo, preparando e analisando dados para prever tendências emergentes e fornecer recomendações para otimizar os resultados.
- Executar tarefas de engenharia de dados, modelagem e análise de negócios.
- Sólidos fundamentos científicos, bem como visão de negócios.

**Todas as nossas vagas são elegíveis para pessoas com deficiência ou reabilitadas**

### Introdução

Como Cientista de dados na IBM, você ajudará a transformar os dados de nossos clientes em valor de negócios tangível, analisando informações, divulgando resultados e colaborando no desenvolvimento de produtos. Trabalhe com as melhores ferramentas visuais e softwares livres da categoria, juntamente com as opções de implementação mais flexíveis e escaláveis. Quer se trate de investigar tendências de pacientes ou padrões meteorológicos, você trabalhará para resolver problemas do mundo real para as indústrias, transformando a forma como vivemos.

### Profissional Requerido e Conhecimento Técnico

• Metodologia de ciência de dados (por exemplo: CRISP-DM)
• Habilidades de engenharia de dados para construir arquiteturas de dados
• Gerenciamento de dados, ciência de dados e bibliotecas matemáticas, como Scikit Learn, Pandas, Scipy, Numpy, TensorFlow, Keras, PyTorch
• Algoritmos de Ciência de Dados de Conhecimento, como Redes Neurais, XBoost, Gradient Boost, Random Forrest, Árvores de Decisão, Regressão Logística, Regressão Linear, etc.
• Visualização de ciência de dados usando bibliotecas como Seaborn, Matplotlib, etc.
• Linguagens de programação: Python, NodeJS
• Ferramentas: Watson Studio, Jupyter Notebook, Jupyter Lab, SN Predictive Intelligence, Pycharm, VS Code, Git.
• Inglês - Fluente

### Profissional Preferencial e Conhecimento Técnico

Experience with ServiceNow platform data model.
ITIL certification

 

### Educação requerida
Diploma de Bacharel

### Educação preferencial
Diploma de Bacharel

### Requisitos de elegibilidade

Todas as nossas vagas são elegíveis para pessoas com deficiência ou reabilitadas

### Sobre a unidade de negócios

A organização IBM Finance é responsável por impulsionar o desempenho e a transformação da empresa.  Somos os administradores financeiros da IBM, fornecendo a estratégia de investimento da IBM, desenvolvendo novos modelos de negócios e mitigando o risco corporativo. Você é apaixonado por criar valor comercial? Junte-se à nossa equipe em contabilidade, planejamento financeiro, precificação, controles de negócios, impostos, tesouraria, desenvolvimento de negócios (aquisições e desinvestimentos) e financiamento global.

### Sua vida na IBM

O que importa para você na procura por seu próximo desafio na carreira? Talvez você queira se envolver em um trabalho que realmente mude o mundo. Que tal um lugar com oportunidades de carreira e desenvolvimento incríveis e diversificadas, onde você pode verdadeiramente descobrir sua paixão? Está procurando uma cultura de abertura, colaboração e confiança na qual todos têm voz? E quanto a tudo isso? Em caso afirmativo, a IBM poderia ser o próximo desafio de sua carreira. Junte-se a nós, não para fazer algo melhor, mas para experimentar coisas que você nunca imaginou. Impacto. Inclusão. Experiências ilimitadas. Faça o seu melhor trabalho sempre.

### Sobre a IBM

A IBM é uma empresa líder em soluções de plataforma e cognitiva em nuvem. Reinventando incansavelmente desde 1911, somos o maior empregador de tecnologia e consultoria do mundo, com mais de 380.000 funcionários atendendo clientes em 170 países. Com o Watson, a plataforma de IA para empresas, impulsionada por dados, estamos construindo soluções baseadas na indústria para problemas do mundo real.

### Declaração de localização

Para obter informações adicionais sobre os requisitos de localização, entre em contato com o recrutador responsável por acompanhar sua candidatura.

### Sendo você na IBM

. A IBM está comprometida em criar um ambiente diversificado e tem orgulho de ser uma empregadora que oferece oportunidades iguais. Todos os candidatos qualificados serão considerados para o cargo, independentemente de raça, cor, religião, sexo, identidade ou expressão de gênero, orientação sexual, origem nacional, genética, gravidez, deficiência, idade, status de veterano ou outras características. A IBM também está comprometida em cumprir com todas as práticas trabalhistas justas relacionadas a status de cidadania e imigração.

https://krb-sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=26059&siteid=5026&PageType=JobDetails&jobid=363557
